static/scss/detail/learning-techniques.scss

```diff
@@ -0,0 +1,75 @@
+// sass-lint:disable mixins-before-declarations
+@import "common";
+
+.learning-techniques-block {
+  padding-top: 70px;
+  padding-bottom: 50px;
```

Comment:
This issue isn't limited to your PR, but there are very large spaces between detail page subsections in the mobile view. Could you standardize these subsections to have a padding of ~25px on the top and bottom for small screens? It might be good to add a common class to all of the subsections besides the header and assign the style to it.

<img width="357" alt="ss 2019-05-02 at 12 33 27 " src="https://user-images.githubusercontent.com/14932219/57091191-99c2ea80-6cd6-11e9-8961-468f05543334.png">
<img width="367" alt="ss 2019-05-02 at 12 37 23 " src="https://user-images.githubusercontent.com/14932219/57091381-105fe800-6cd7-11e9-87d6-ffab24648291.png">

Reply:
added padding bottom-top : 40px. I am thinking, After rendering Enroll #143 section  in b/w "Learning Outcomes" & "Learning Techniques" it will be better to view but can be improve at the end of page.